### PR TITLE
Test search

### PR DESCRIPTION
**Pull request by LucyGwilliamAdmin**

### CODE REVIEW

File: _includes/assets/js/accessibility.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ var accessibilitySwitcher = function() {
       'class': 'nav-link contrast contrast-' + contrast
     }).html($('<a />').attr({
       'href': 'javascript:void(0)',
-      'onClick': 'ga("send", {"hitType": "event", "eventCategory": "Change setting", "eventAction": "Change contrast setting", "eventLabel": "Change to " +contrast+ " contrast"})',
+      'onClick': 'ga("send", "event", "Change setting", "Change contrast setting", "Change to " +contrast+ " contrast")',
```
Comment (dougmet) [inline _includes/assets/js/accessibility.js]: out of interest what does this do?

File: _includes/assets/js/search.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  var escapeRegExp = function(str) {
+    return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/gi, "\\$&");
+  };
+
+  if($('#main-content').hasClass('search-results')) {
+
+    var results = [],
+        that = this,
+        searchString = unescape(location.search.substring(1)).replace("q=", "");
```
Comment (dougmet) [inline _includes/assets/js/search.js]: am I right in thinking this is the only change here?

**Comment by dougmet** [comment]
I like this. If we can get it through open-sdg quickly enough would it best there?

File: _includes/assets/js/accessibility.js (hunk 1 above)
Comment (LucyGwilliamAdmin) [reply _includes/assets/js/accessibility.js]: Allows us to track when someone changes the contrast they are viewing in. Not actually anything to do with the search in this case.

File: _includes/assets/js/search.js (hunk 2 above)
Comment (LucyGwilliamAdmin) [reply _includes/assets/js/search.js]: Yes

**Comment by LucyGwilliamAdmin** [comment]
> I like this. If we can get it through open-sdg quickly enough would it best there?

Definitely think it will be useful for others who are using Google Analytics (I know Brock mentioned that Armenia are/thinking about it)

File: _includes/assets/js/accessibility.js (hunk 1 above)
Comment (dougmet) [reply _includes/assets/js/accessibility.js]: and changing `"hitType"` etc?

Comment (LucyGwilliamAdmin) [reply _includes/assets/js/accessibility.js]: I followed two different tutorials and the first one didn't seem to work so I'm trying the second one.

It doesn't seem to have broken anything but I can't actually test whether the event tracking works unless it goes on to the live site.

Comment (LucyGwilliamAdmin) [reply _includes/assets/js/accessibility.js]: I actually changed it before I changed it on this branch, that's why the history isn't here

Comment (dougmet) [reply _includes/assets/js/accessibility.js]: yes I see you can call the function two ways so it should be equivalent: https://developers.google.com/analytics/devguides/collection/analyticsjs/events

Comment (dougmet) [reply _includes/assets/js/accessibility.js]: Something is actually broken on the live site: 
image

Comment (dougmet) [reply _includes/assets/js/accessibility.js]: when you click on the contrast something is wrong in this call to `ga` so I doubt it's getting through to Google Analytics

Comment (LucyGwilliamAdmin) [reply _includes/assets/js/accessibility.js]: Could it be where I've tried to use the contrast variable in the call?

Comment (dougmet) [reply _includes/assets/js/accessibility.js]: This is broken in `open-sdg` too. Would it be possible to remove any changes to `accessibility` and we'll open a separate bug fix branch/issue? Then we can merge in the search changes.

Comment (LucyGwilliamAdmin) [reply _includes/assets/js/accessibility.js]: That sounds like a good idea, I'll remove them now

**Pull request merged**